product_pricelist_direct_print/i18n/es.po

```diff
@@ -95,7 +95,7 @@ msgstr "<strong>Descripción</strong>"
 #. module: product_pricelist_direct_print
 #: model_terms:ir.ui.view,arch_db:product_pricelist_direct_print.report_product_pricelist_document
 msgid "<strong>List Price</strong>"
-msgstr "<strong>Precio tarifa</strong>"
+msgstr "<strong>Precio de venta</strong>"
```

Comment:
What's wrong with the current term?

Reply:
In any moment, it was "Precio de coste", and I suggest this as better term.

Reply:
Thinking twice, it's true that this field is showing the specific price of this pricelist, so it can be more accurate to say "tarifa", but the proper Spanish form is `Precio de tarifa` (with `de`). The other thing is not correct, although tolerated for abbreviating things. Also calling it simply `Precio` can be another option. Which one do you prefer?

Reply:
We can show 3 distinct prices with this module:
https://github.com/OCA/product-attribute/blob/8a43eda8f5c47fc8d7c68714bd146f21c55dc11a/product_pricelist_direct_print/views/report_product_pricelist.xml#L57-L65
https://github.com/OCA/product-attribute/blob/8a43eda8f5c47fc8d7c68714bd146f21c55dc11a/product_pricelist_direct_print/views/report_product_pricelist.xml#L84-L94
So I prefer "Precio tarifa" without "de" to get a little abbreviation.